vufs/src/main/java/com/stnetix/ariaddna/vufs/DTO/ChunkFile.java

```diff
@@ -0,0 +1,41 @@
+package com.stnetix.ariaddna.vufs.DTO;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Describe the chunk of file. Field url contain the place where this chunk stored.
+ * Used in class {@link VirtualFile} as metadata of part of file.
+ * Field urls contain collection of url to the chunk.
+ * verifyHash is MD5 hash string of the chunk byte array.
+ */
+public class ChunkFile implements Serializable {
+    private UUID uuid;
+    private List<String> urls;
+    private String verifyHash;
+
```

Comment:
I think, that we need at least two constructors here.
- With one parameter - verifyHash. It will be calculated outside just once during chunk creation and passed to the constructor. Also inside it a UUID must be generated.
- With two parameters: verifyHash and UUID